cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/RunCommand.java

```diff
@@ -86,6 +86,10 @@ public class RunCommand implements BLauncherCmd {
             "when run is used with a source file or a module.")
     private Boolean observabilityIncluded;
 
+    @CommandLine.Option(names = "--remote-management", description = "package management service in the " +
```

Comment:
```suggestion
    @CommandLine.Option(names = "--remote-management", description = "enable management service in the " +
```
`package` wording can get confused with the Ballerina package.